### PR TITLE
Create pdf from json

### PR DESCRIPTION
**Pull request by sammjainn**
For one tweet, it looks like this:
Capture

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -42,45 +45,56 @@ Here's a video explaining the git branch in short [![tutorial](https://img.youtu
...
+
+## Logo 🎨
 
 **Where to make the logo from?**
+
 - May use free platforms like Canva, Gimp, or Inkspace
 - Use photoshop (if you already have the software)
 
 **How to share the logo I made?**
-- Comment on the issue **[#4 "We need a logo"](https://github.com/twindle-co/twindle/issues/4)** raised by johnjacobkenny
```
Comment (UnevenCoder) [inline docs/FAQ.md:82]: This was better 😅

**Review by UnevenCoder — APPROVED** [review]
Left a review also I suggest doing multiple prs for different rather than one .. will make the review process easy and approval will be fast 😅🙂

**Comment by proful** [comment]
Wow awesome 😊👏

**Review by tolgaerdonmez — COMMENTED** [review]
Good approach creating pdfs

**Comment by proful** [comment]
2 approval?

**Comment by tolgaerdonmez** [comment]
> 2 approval?

I've just commented, or I didn't understand what you mean :)

File: docs/FAQ.md (hunk 1 above)
Comment (sammjainn) [reply docs/FAQ.md:82]: I made changes then discarded. This is just because of pulling from upstream.

**Comment by johnjacobkenny** [comment]
@sammjainn I tried running the code and I am getting this error.
image

File: pdf/index.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    twitterHandle: 'naval',
+    image:
+      'https://pbs.twimg.com/profile_images/1256841238298292232/ycqwaMI2_400x400.jpg',
+    createAt: '2018-05-31',
+    tweet: 'How to Get Rich (without getting lucky):'
+  }
+];
+
+// setting available fonts
+var fonts = {
```
[the remark this answers is not in the document]
Comment (johnjacobkenny) [reply pdf/index.js:14]: Also please eventually move this file to `playground/cli`

[the remark this answers is not in the document]
Comment (sammjainn) [reply pdf/index.js:14]: okay, making these changes and committing.

**Comment by sammjainn** [comment]
@johnjacobkenny I didn't put up everything (package-json etc), you'll need to npm init, then install module pdfmake. Should I upload everything?

**Comment by johnjacobkenny** [comment]
Yes please include everything required to run the code

File: pdf/index.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    normal: 'ZapfDingbats'
+  }
+};
+
+var PdfPrinter = require('pdfmake');
+var printer = new PdfPrinter(fonts);
+var fs = require('fs');
+
+// document
+var docDefinition = {
```
Comment (johnjacobkenny) [inline pdf/index.js:46]: Could be `const`, please look through other instances too :)

Comment (sammjainn) [reply pdf/index.js]: done

**Review by johnjacobkenny — APPROVED** [review]

**Pull request merged**